inc/Runner.php

```diff
@@ -12,7 +12,11 @@
 use WP_Filesystem_Base;
 
 /**
- * Project class.
+ * Project update runner.
```

Comment:
I still have no clue what this should tell someone who wants to know what this class actually does but seems like you're not going to change it. 🤷‍♂️

Reply:
There's a longer explanation in the description below 🤔 

If that still is unclear for you then you're welcome to change this yourself. I can't read minds 🙂

Reply:
Why does the "longer" description have to be below? 🤔 

"Project update runner." simply doesn't match what core's handbook is advising for an informative summary of a class.

Reply:
Never said we're following the core standards for docs 😅 One's gonna read the whole docblock, so this is more bikeshedding at this point than anything else.

But anyway, I can update it to something like `Update runner class to update a project's strings given a specific loader and an updater.` if that sounds better.